event_sale_session/tests/test_event_sale_session.py

```diff
@@ -54,22 +57,6 @@ def test_sale_session(self):
         orders = self.env["sale.order"].search(action["domain"])
         self.assertIn(self.order, orders)
 
-    def test_sale_order_line_session_onchange_autocomplete(self):
-        """Test that session is automatically filled and or unset on form"""
-        form = Form(self.order)
-        line = form.order_line.new()
-        line.product_id = self.product
-        # Case 1: The event is a session event, but has multiple sessions
-        line.event_id = self.event
-        self.assertFalse(line.event_session_id)
-        # Case 2: The event is a session event with only 1 session
-        (self.event.session_ids - self.session).active = False
-        line.event_id = self.event
-        self.assertEqual(line.event_session_id, self.session)
-        # Case 3: The event is not a session event, session should be unset
-        line.event_id = self.env.ref("event.event_0")
-        self.assertFalse(line.event_session_id)
-
```

Comment:
Why removing this whole test?

Reply:
Onchange is only applied when you are in form view

Reply:
Sorry, I still don't understand the reason to remove it 😓 are you facing an issue with this test while implementing the new feature?